src/søknad/barna-dine/LeggTilBarn.tsx

```diff
@@ -0,0 +1,115 @@
+import React, { useState } from 'react';
+import { Hovedknapp } from 'nav-frontend-knapper';
+import { Normaltekst, Undertittel } from 'nav-frontend-typografi';
+import useSøknadContext from '../../context/SøknadContext';
+import { differenceInYears } from 'date-fns';
+import { RadioPanel } from 'nav-frontend-skjema';
+import { formatDate, formatDateFnr } from '../../utils/dato';
+import LeggTilBarnFødt from './LeggTilBarnFødt';
+import LeggTilBarnUfødt from './LeggTilBarnUfødt';
+import { IBarn } from '../../models/person';
+const uuidv4 = require('uuid/v4');
+
+interface Props {
+    settÅpenModal: Function;
+}
+
+const LeggTilBarn: React.FC<Props> = ( { settÅpenModal }) => {
+  const { søknad, settSøknad } = useSøknadContext();
+  const [barnDato, settBarnDato] = useState<Date>(new Date());
+  const [født, settBarnFødt] = useState("");
+  const [navn, settNavn] = useState("Barn");
+  const [personnummer, settPersonnummer] = useState("");
+  const [boHosDeg, settBoHosDeg] = useState("");
+
+  const settDato = (date: Date | null): void => {
+    date !== null && settBarnDato(date);
+  };
+
+  const settFødt = (event: any) => {
+    settBarnFødt(event.target.value);
+  }
+
+  const settBo = (event: any) => {
+    settBoHosDeg(event.target.value);
+  }
+
+  const tilbakestillFelt = () => {
+    settBarnDato(new Date());
+    settNavn("Barn");
+    settPersonnummer("");
+    settBoHosDeg("");
+  }
+
+  const leggTilBarn = () => {
+    const fødselsnummer = barnDato && personnummer ? formatDateFnr(barnDato) + personnummer : "";
+
+    const barn = {
+      fnr: fødselsnummer,
+      personnummer: personnummer,
+      alder: differenceInYears(new Date(), barnDato),
+      navn: navn,
+      fødselsdato: formatDate(barnDato),
+      harSammeAdresse: boHosDeg === "ja",
+      ufødt: født === "nei",
+      lagtTil: true,
+      id: uuidv4(),
+    }
+
+    const nyBarneListe = [...søknad.person.barn, barn];
+
+    settSøknad({...søknad, person: {...søknad.person, barn: nyBarneListe}});
+
+    settÅpenModal(false);
+  }
+
+  return (
+        <div className="legg-til-barn">
+          <Undertittel>Legg til barn</Undertittel>
+
+          <Normaltekst className="spørsmål">Er barnet født?</Normaltekst>
```

Comment:
<Element/> for bold tekst i stedet for å ha en css klasse med `font-weight: bold` :) 
Og intl senere :) når komponentene er klare!